ui/src/components/ClientsTable/index.jsx

```diff
@@ -95,17 +102,19 @@ export default class ClientsTable extends Component {
         onPageChange={onPageChange}
         renderRow={({ node: client }) => (
           <TableRow key={client.clientId}>
-            <TableCell padding="dense">
-              <TableCellItem
-                dense
-                button
-                component={Link}
-                to={`/auth/clients/${encodeURIComponent(client.clientId)}`}>
-                <ListItemText primary={client.clientId} />
-                <LinkIcon size={iconSize} />
-              </TableCellItem>
+            <TableCell size="small">
```

Comment:
Instead of adding `size="small"` to every table cell, I think we can just do `<Table size="small">...</Table>`. We can expose a `size` prop and have it default to `small`. That way, we don't really need to do anything except remove `size="small"` from all table cells.